modules/update-upgrading-sro.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-clusters-sro.adoc
+
+:_content-type: PROCEDURE
+[id="updating-clusters-sro_{context}"]
+= Running a preflight check for the SRO
+
+The following is an example procedure for checking the compatibility of a kernel module version before upgrading a cluster.
+
+.Prerequisites
+
+* You have a running {product-title} cluster.
+* You installed the OpenShift CLI (oc).
+* You are logged into the OpenShift CLI as a user with cluster-admin privileges.
+* You installed the SRO.
+
+.Procedure
+
+. Create the following CRD and save the YAML as `prevalidation.yaml`.
++
+[source,yaml]
+----
+apiVersion: sro.openshift.io/v1beta1
+kind: PreflightValidation
+metadata:
+ name: preflight
+ namespace: preflight
+spec:
+ updateImage: quay.io/openshift-release-dev/ocp-release@sha256:f7f252c39b64601c8ac3de737a584ba4f6016b1f4b17801d726ca2fd15492878 <1>
+----
+<1> Specify the name of the update image here.
++
+. Start the validation check by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f prevalidation.yaml
+----
+
+.Verification
+. Check the status of the CR by running the following command:
++
+[source,terminal]
+----
+$ oc describe preflightvalidations.sro.openshift.io/v1beta1 preflight
+----
++
+.Example output
+The following is an example output in which `simple-oot` is a `SpecialResource` CR that has been deployed on the cluster.
++
+[source,terminal]
+----
+Status:
+  Cr Statuses:
+    Last Transition Time:  2022-08-02T08:48:45Z
+    Name:                  simple-oot
+    Status Reason:         Verification successful, all driver-containers for the next kernel version are present
+    Verification Stage:    Image
+    Verification Status:   True
+Events:                    <none>
+----
++
+The possible CR statuses are as follows:
++
+--
+`True`:: The SRO CR will upgrade without kernel compatibility issues.
+`False`:: There is an issue with kernel compatibility for the CR. The Status Reason field provides additional information on this.
+`Error`:: The status check was not completed due to an internal error. The Status Reason field provides additional information on debugging this.
+`Unknown`:: The check has not yet verified the status of the CR. This may be because the process has not yet reached the specific CR or not enough time has elapsed for the check to complete.
+--
++
+The possible verification statuses are as follows:
++
+--
+`True`:: The image exists and is compatible or the image does not exist but there is a BuildConfig in place.
```

Comment:
To enhance the readability of this sentence and facilitate localisation, you could include a comma to separate the two aspects of this sentence. For example:

> The image exists and is compatible, or the image does not exist but there is a BuildConfig in place.

Reply:
OK